assets/sass/components/global/_googlesitekit-header.scss

```diff
@@ -184,4 +209,12 @@
 		display: flex;
 		justify-content: flex-end;
 	}
+
+	.googlesitekit-header__children > div:not(:first-child) {
+		margin-left: $grid-gap-phone;
+
+		@media (min-width: $bp-desktop) {
+			margin-left: $grid-gap-desktop;
```

Comment:
Can we do this with `gap` on the `googlesitekit-header__children` instead?

Here's with the margin
![image](https://user-images.githubusercontent.com/1621608/148595654-623da542-683f-4de0-aff6-b75ed5703c3c.png)

Here's with `gap`
![image](https://user-images.githubusercontent.com/1621608/148595662-b72bd17c-febf-41dc-adbf-92ec35aab808.png)